skopt/gp_opt.py

```diff
@@ -95,7 +96,9 @@ def gp_minimize(func, dimensions, base_estimator=None, acq="LCB", xi=0.01,
         Number of calls to `func`.
         If `n_random_starts` > 0, `n_calls - n_random_starts`
         additional evaluations of `func` are made that are guided
-        by the `base_estimator`.
+        by the `base_estimator`. If `x0` is provided but not `y0`
+        then `n_calls - len(x0) - n_random_starts` evaluations
+        are made instead of `n_calls - n_random_starts` .
```

Comment:
In total `n_calls` evaluations are always made. 

We should rather say something like: "If x0 is provided but not y0, then these points are first evaluated, followed by `n_random_starts` random initialization points,  and finally `n_calls - len(x0) - n_random_starts` subsequent evaluations as guided by the surrogate model." (Text can be improved, but you get the idea)